IDE/Espressif/ESP-IDF/examples/template/components/wolfssl/include/user_settings.h

```diff
@@ -208,7 +538,7 @@
 #define USE_FAST_MATH
 
 /*****      Use SP_MATH      *****/
-/* #undef USE_FAST_MATH          */
+/* #undef  USE_FAST_MATH          */
```

Comment:
suggest you either snip out the added space after `#undef`, or snip out a space afterwards to get the `*/` to line up.

Reply:
definitely!